src/bluehawk/parser/parser.test.ts

```diff
@@ -441,6 +466,45 @@ this is not bluehawk markup
       );
     });
 
+    it("rejects extra block comment end that straddles inside of comment", () => {
+      const test_string = `:block-command-start: */ */
+chunky chunky
+/* :block-command-end:`;
+      const result = lexer.tokenize(test_string);
+      expect(result.errors.length).toBe(0);
+      parser.input = result.tokens;
+      parser.annotatedText();
+      expect(parser.errors[0].message).toBe(
+        "1:23(22) blockCommandUncommentedContents: After BlockCommentEnd, expected Newline but found BlockCommentEnd"
+      );
+    });
+
+    it("rejects extra block comment start that straddles inside of comment", () => {
+      const test_string = `:block-command-start: */
+chunky chunky
+/* /* :block-command-end:`;
+      const result = lexer.tokenize(test_string);
+      expect(result.errors.length).toBe(0);
+      parser.input = result.tokens;
+      parser.annotatedText();
+      expect(parser.errors[0].message).toBe(
+        "3:4(42) blockComment: After BlockCommentStart, expected BlockCommentEnd but found CommandEnd"
+      );
+    });
+
+    it("rejects block comment in attribute list", () => {
+      const test_string = `:block-command-start: { att1, att2, */, att3}
+chunky chunky
+/* :block-command-end:`;
+      const result = lexer.tokenize(test_string);
+      expect(result.errors.length).toBe(0);
+      parser.input = result.tokens;
+      parser.annotatedText();
+      expect(parser.errors[0].message).toBe(
+        "1:23(22) attributeList: After AttributeListStart, expected AttributeListEnd but found BlockCommentEnd"
```

Comment:
Does it actually support something like this?
```
/* :block-command-start: { */
  /* "att1": 1, */
  /* "att2": 2 */
/* } */
```

If not, don't worry about it for now, but come to think of it, I would kind of expect this to work, especially since we went through the trouble of adding those tokens to the attribute list mode.

Reply:
So right not this won't work as the attribute list lexer mode is popped by both new lines (behavior before this PR) and comment end tokens (this is added by this PR). I think we could make this work, but it would involve some possibly tricky changes in how the attribute list lexer mode is exited.

Agree that in an ideal world this would work. [I have captured this as a feature request in trello](https://trello.com/c/9IlLsA4Q/77-support-multi-line-attribute-list-declerations)